.gitignore

```diff
@@ -18,3 +19,9 @@ testfm.egg-info/
 #Packaging
 build/
 dist/
+
+# Dynaconf local files
+/.secrets.yaml
+/.secrets_*.yaml
+/testfm.sample.yaml
```

Comment:
Why is the sample being ignores by git?

Reply:
Now, ignoring all changes from conf/*  and excluding sample by git, will that work?